language/move-prover/lab/data/vector-theories/notebook.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "id": "sixth-exhaust",
+   "id": "streaming-dallas",
```

Comment:
what do these ids like "streaming-dallas", 'legitimate-funtionality", etc. mean?

Reply:
I have no idea. They are automatically generated by Jupyterbook. Looking at those raw files isn't useful, you need to open them in the browser with `./notebook.sh`.